@theme/components/Navbar/Navbar.tsx

```diff
@@ -1,12 +1,11 @@
 import * as React from "react";
 import styled from "styled-components";
-import { useThemeConfig } from "@theme/hooks/useThemeConfig";
-import { LanguagePicker } from "@theme/i18n/LanguagePicker";
-import { useI18n, useTranslate } from "@portal/hooks";
+import { useThemeConfig, useThemeHooks } from "@redocly/theme/core/hooks";
+import { LanguagePicker } from "@redocly/theme/components/LanguagePicker/LanguagePicker";
 import { slugify } from "../../helpers";
-import { Link } from "@portal/Link";
-import { ColorModeSwitcher } from "@theme/components/ColorModeSwitcher/ColorModeSwitcher";
-import { Search } from "@theme/components/Search/Search";
+import { Link } from "@redocly/theme/components/Link/Link";
```

Comment:
If we have to change every instance of this, there might be a few new ones that got added after this branch was forked.